docs/developer/git.md

```diff
@@ -0,0 +1,104 @@
+# Git help
+
+## Creating new branches
+
+1. Create a unique branch name tracking `origin/master`
+
+```bash
+git checkout -b <users/alias/feature-branch-name> origin/master
+```
+
+2. Use git status to verify the current branch and modified files.
+
+```bash
+git status
+```
+
+### Recommended practices
+
+* Use all lower cased hyphen-delimited branch names for readability
+* Format as <users/alias/branch-name>. For example, users/jdoe/task-issue-title
+* Create as new branch for each work issue using the `-b` argument.
+* Commit messages should use lowercase imperative present tense. Use commands. Instead of “I added tests for ...” use “add tests for ...”
+* [Rebase, don't merge](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
+
+## Common commands
+
+All "packages" share common script names for consistency along with their own documentation guides.
+
+To clean all packages.
+
+```bash
+lerna clean
+```
+
+When package files change the symbolic links will require updating.
+
+```bash
+lerna bootstrap
+```
+
+To run all tests on all packages.
+
+```bash
+lerna run test
+```
+
+To run all tests on a single package.
+
+```bash
+cd packages/package-name
+npm run test
+```
+
+Most packages have a *watch* command that will rebuild the package's distribution when a file changes. This process can be useful when doing development across multiple packages.
+
+```bash
+npm run watch
+```
+
+## Common housekeeping commands
```

Comment:
I don't think we need this documentation; it feels like clutter to me. There's a ton of info out on the web regarding git and I think it's a pre-req. IMO, some of this can move to another section, but the GIT page needs to go